### ISSUE TITLE
Properly setting up a fork of this project

### ISSUE THREAD

**Issue by opswhisperer** [danielgindi/Charts#1133]
As expected there are a ton of syntax changes in xc8 and Swift 2.3 (not even trying 3 yet). I figured I'd fork this and send a pull request. I've forked your repo and oddly only a few changes were needed to build it. However in my project the Charts framework has nearly a hundred issues.

I must be missing something, I pointed my Podfile at my (updated) fork and all the issues still exist in my project even thought the Charts project compiles.

Do you have notes on getting setup to contribute?

**Comment by opswhisperer** [comment]
Nevermind I got this working. Unfortunatley there is way too much that has changed for me to be helpful. I've been fixing easy syntax changes for over an hour and there are hundreds more to go. The problem is its not all syntax, there are code changes required as well. I don't know enough about the internals of this project to do mass edits.  

I don't suppose you have an ETA for supporting Xcode8 & Swift 2.3?  (By thew way the README says Swift 3, I'm don't think thats correct)

**Comment by opswhisperer** [comment]
The code "converter" does not work in beta1 - it made things much worse. Just FYI

**Comment by liuxuan30** [comment]
well, I am in WWDC and aware of a lot of changes, so this may be a huge change; it says supports swift 3.0 probably means we have removed the c style loops and enums :) not others

**Comment by opswhisperer** [comment]
It will be, I was hoping to help but there's just too much to change.

**Comment by liuxuan30** [comment]
:( I was just downloading XC 8 and grasp an engineer to take a look

**Comment by opswhisperer** [comment]
I figured out some of the problem. There is a build setting SWIFT_VERSION when you run the converter for 2.3 this gets set. However when cocoapods pulls in the framework it does not respect that setting so when I added Charts back to my project it was validating for Swift 3.

I'm sending a PR for swift2.3 branch of fix the swift version problem I added this to my Podfile
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['SWIFT_VERSION'] = '2.3'
    end
  end
End
```
If its not a swift pod it still gets added but ignored, if it its a Swift pod then it sets the `Use Legacy Swift Language Version` build setting to `Yes`
